src/pokemon_mood.c

```diff
@@ -1,24 +1,27 @@
-#include "global.h"
-#include "pokemon.h"
 #include "pokemon_mood.h"
+
+#include "global.h"
+
 #include "constants/items.h"
 
+#include "pokemon.h"
+
 static const s8 sMoodModifiers[] = {
-    [MON_MOOD_MODIFIER_LEVEL_UP_IN_BATTLE] =   40,
-    [MON_MOOD_MODIFIER_1]                  =    8,
-    [MON_MOOD_MODIFIER_2]                  =   50,
-    [MON_MOOD_MODIFIER_LEARN_TMHM]         =   40,
-    [MON_MOOD_MODIFIER_4]                  =  -80,
-    [MON_MOOD_MODIFIER_SURVIVED_PSN]       =  -50,
+    [MON_MOOD_MODIFIER_LEVEL_UP_IN_BATTLE] = 40,
+    [MON_MOOD_MODIFIER_1]                  = 8,
+    [MON_MOOD_MODIFIER_2]                  = 50,
+    [MON_MOOD_MODIFIER_LEARN_TMHM]         = 40,
+    [MON_MOOD_MODIFIER_4]                  = -80,
+    [MON_MOOD_MODIFIER_SURVIVED_PSN]       = -50,
     [MON_MOOD_MODIFIER_6]                  = -100,
-    [MON_MOOD_MODIFIER_7]                  =   30,
+    [MON_MOOD_MODIFIER_7]                  = 30,
```

Comment:
@lhearachel how would i get these numbers to right-align?

Reply:
As above, I'm pretty sure clang-format doesn't support right-alignment like that; I remember that being an issue in platinum